scrapy/loader/__init__.py

```diff
@@ -35,8 +35,9 @@ def __init__(self, item=None, selector=None, response=None, parent=None, **conte
         self._local_values = defaultdict(list)
         # Preprocess values if item built from dict
         # Values need to be added to item._values if added them from dict (not with add_values)
-        for field_name, value in item.items():
-            self._values[field_name] = self._process_input_value(field_name, value)
+        if type(item) == dict:
```

Comment:
```suggestion
        if isinstance(item, dict):
```
Just trying the _suggestion_ feature 😄